spec/support/examples/idempotent_resource.rb

```diff
@@ -1,9 +1,9 @@
 shared_examples 'an idempotent resource' do
   it 'should run without errors' do
-    apply_manifest(@manifest, :expect_changes => true)
+    apply_manifest(@manifest, :catch_failures => true)
```

Comment:
This change was not necessary, however in a lot of our other modules tests we are using `:catch_changes` and `:catch_failures` rather than `:expect_changes` and `:expect_failures` and I was changing below from `:expect_changes => false` to `:catch_changes => true` and thought it best that I change this also for the sake of consistency.